docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -106,159 +134,160 @@ This creates a tarball of the built collection in the current directory which ca
 
 * ``*``: Matches everything
 * ``?``: Matches any single character
-* ``[seq]``: Matches and character in seq
-* ``[!seq]``:Matches any character not in seq
+* ``[seq]``: Matches any character in sequence
+* ``[!seq]``:Matches any character not in sequence
 
-For example, if you wanted to exclude the :file:`sensitive` folder within the ``playbooks`` folder as well any ``.tar.gz`` archives you
-can set the following in your ``galaxy.yml`` file:
+For example, to exclude the :file:`sensitive` folder within the ``playbooks`` folder as well any ``.tar.gz`` archives, set the following in your ``galaxy.yml`` file:
 
 .. code-block:: yaml
 
      build_ignore:
      - playbooks/sensitive
      - '*.tar.gz'
 
+For more information on the :file:`galaxy.yml` file, see :ref:`collections_galaxy_meta`.
+
 .. note::
-     This feature is only supported when running ``ansible-galaxy collection build`` with Ansible 2.10 or newer.
+     The ``build_ignore`` feature is only supported with ``ansible-galaxy collection build`` in Ansible 2.10 or newer.
 
-.. _collection_versions:
+.. _trying_collection_locally:
 
-Collection versions
-===================
+Preparing to publish your collection
+====================================
 
-Each time you publish your collection, you create a new version. Once you publish a version of a collection, you cannot delete or modify that version. Ensure that everything looks okay before publishing. The only way to change a collection is to release a new version. The latest version of a collection (by highest version number) will be the version displayed everywhere in Galaxy or Automation Hub; however, users will still be able to download older versions.
+Each time you publish your collection, you must create a :ref:`new version <collection_versions>` on the distribution server. After you publish a version of a collection, you cannot delete or modify that version. To avoid unnecessary extra versions, check your collection for bugs, typos, and other issues locally before publishing:
 
-Collection versions use `Semantic Versioning <https://semver.org/>`_ for version numbers. Please read the official documentation for details and examples. In summary:
+#. Install the collection locally.
+#. Review the locally installed collection before publishing a new version.
 
-* Increment major (for example: x in `x.y.z`) version number for an incompatible API change.
-* Increment minor (for example: y in `x.y.z`) version number for new functionality in a backwards compatible manner (for example new modules/plugins, parameters, return values).
-* Increment patch (for example: z in `x.y.z`) version number for backwards compatible bug fixes.
+Installing your collection locally
+----------------------------------
 
+You can install your collection locally in two ways:
 
-.. _trying_collection_locally:
+  * from the tarball
+  * from your git repository
```

Comment:
@acozine Suggestion to capitalize and add a period at the end of each sentence:
- From the tarball.
- From your git repository.

Reply:
Addressed in 60a5bfd89a256ebc871c5ecb87d7539c1c248c56. Thanks @lmalivert.